tensorrt_llm/_torch/locality_domain_utils.py

```
@@ -0,0 +1,565 @@
+
+def is_locality_domain_supported() -> bool:
+    """
+    Check if LOCALITY_DOMAIN localization is supported on this system.
+
+    Returns:
+        bool: True if LOCALITY_DOMAIN localization is supported, False otherwise.
+    """
+    try:
+        handle = _tbr.LocalizationHandle()
+        return handle.supports_localization()
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)